transcom/mymove
# B 18492 Prime API sitAddressUpdate Deprecation Documentation Updates

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a889878)

## Summary

As we officially transition to having the shipment's change of `destination_address` drive the change of destination SIT service items' `sit_destination_final_address`, we need to let the Prime know this by updating the API documentation.

This PR does the following:
- updates Prime API documentation to notify them that the endpoint `createSITAddressUpdateRequest` will be deprecated 6 months from now
- in the description of the details, notified them of the change and what endpoints can be used to update the `sit_destination_final_address` after the endpoint is deprecated

### How to test

1. Fire up the client & server
2. Open up [Swagger docs](http://officelocal:3000/swagger-ui/prime.html#/sitAddressUpdate/createSITAddressUpdateRequest) to see the changes
